### DISCUSSION TITLE
Parallel implementation of Kalman filters using JAX associative_scan

### DISCUSSION THREAD

**Discussion by AdrienCorenflos** [jax-ml/jax#5353] [Show and tell]
Hi all,

Thought I'd share this cool (obviously I'm gonna say that as I wrote the code :)) use of JAX:

https://github.com/EEA-sensors/sequential-parallelization-examples/
The code follows the paper "Temporal Parallelization of Bayesian smoothers" published this month in this month IEEE TAC issue.

Please check the announcement tweet:
https://twitter.com/simosarkka/status/1347521322486812675

**Comment by mattjj** [comment]
This is super cool, thanks for sharing! I used to work on Bayesian time series methods, so it warms my heart to see JAX supporting this kind of work!

**Reply by AdrienCorenflos** [reply]
Stay tuned then, we should have some more coming :smiley:
